build-scripts/Dockerfile.windows-x64

```diff
@@ -1,4 +1,4 @@
-FROM rust:stretch as build
+FROM rust:buster as build
```

Comment:
It looks like Windows builds succeed if this uses `rust:bullseye`. The `gcc-mingw-w64-x86-64` package version currently used by the buster image runs into an internal compiler error.

Reply:
Cool, I'll switch to `bullseye` in these docker files and reenable to windows output.